### ISSUE TITLE
Add selected option for select boxes (under "forms")

### ISSUE THREAD

**Issue by seanredmond** [NYPL/design-toolkit#196]
One option is a select box should be selected by default (`<option selected="selected">...</option>`). The select box pattern (and others that use select boxes) should reflect this

(See [H91: Using HTML form controls and links](https://www.w3.org/TR/WCAG20-TECHS/H91.html#H91-subhead-14))

**Comment by ricardoom** [comment]
We'll need this on the search results and in the omni-search yeah?

**Comment by ricardoom** [comment]
I'll assign this to myself for the time being.

**Comment by EdwinGuzman** [comment]
Yes, all the `<select>` forms.
